client/my-sites/hosting/hosting-activate.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import MainComponent from 'components/main';
+import HeaderCake from 'components/header-cake';
+import EligibilityWarnings from 'blocks/eligibility-warnings';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { initiateThemeTransfer } from 'state/themes/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+const HostingActivate = ( { initiateTransfer, siteId, siteSlug, translate } ) => {
+	const backUrl = `/hosting-config/${ siteSlug }`;
+
+	const transferInitiate = () => {
+		initiateTransfer( siteId, null, null );
```

Comment:
<img width="1263" alt="Screen Shot 2019-11-27 at 1 47 11 PM" src="https://user-images.githubusercontent.com/1270189/69761553-91bc1380-111c-11ea-8268-1c6dfca6e5d6.png">

During transfer looks like automated_transfer events are being called incorrectly with both the "theme" and "plugins" context. Before shipping, let's try to tidy these to pass though a new context of "hosting" or similar name.

Reply:
I'll have the team cleanup the unrelated JS error in a different diff :)

Reply:
To show analytics locally run `localStorage.debug = 'calypso:analytics*'` in console